uint32 id
string finger_type
float64 length
float64 width
Bone[4] bones